tests/settings.py

```diff
@@ -92,3 +93,6 @@
     from local_settings import *
 except ImportError:
     pass
+
+STATICFILES_DIRS = ( os.path.join(BASE_DIR,'test_project'),)
```

Comment:
why are you adding this? Is it needed? I don't think it should be required so it should be possible to remove it.

Reply:
:+1: